models/metadata.go

```diff
@@ -9,4 +9,5 @@ import (
 type Metadata interface {
 	GetResource() *common.Resource
 	GetProperties() *facets.Properties
+	GetLineage() *facets.Lineage
```

Comment:
@StewartJingga This would not be applicable to every resource right? Like user.

Reply:
yes, but right now it is the only way Columbus sink is able to fetch lineage from our varies models.

The other way is to typecasting for each types which im not sure is scalable.

Reply:
@ravisuhag another approach is to have another interface for lineage and typecheck for that interface in Columbus.
```
type HasLineage interface {
  GetLineage() *facets.Lineage
}
```

Reply:
I think let's create another interface for lineage check instead of expecting all models to have it.